mithril-signer/src/runtime/runner.rs

```diff
@@ -543,8 +561,10 @@ mod tests {
             None,
             slog_scope::logger(),
         ));
+        let block_range_root_retriever = Arc::new(MockBlockRangeRootRetrieverImpl::new());
         let cardano_transactions_builder = Arc::new(CardanoTransactionsSignableBuilder::new(
             transaction_importer,
+            block_range_root_retriever,
```

Comment:
Same remark as the one in `mithril-aggregator/src/dependency_injection/builder.rs`
```suggestion
            Arc::new(MockBlockRangeRootRetrieverImpl::new()),
```